frontend/src/components/Psychologist/Patients/AddEditPatient.jsx

```diff
@@ -59,6 +61,10 @@ const AddEditPatient = () => {
       .catch(() => window.scrollTo(0, 0));
   };
 
+  const today = new Date();
+  const maxPatientDateOfBirth = new Date(today.setFullYear(today.getFullYear() - 100));
+  const minPatientDateOfBirth = new Date(today.setFullYear(today.getFullYear() + 90));
```

Comment:
Are you sure about those ? 
I would have say 
min: - 100
max: -18 ?

Reply:
i reverted it